### ISSUE TITLE
Change hard-coded logs to translated.

### ISSUE THREAD

**Issue by derrix060** [derrix060/onedriveClient#24]
This is not something urgent but will be good once the project starts to get big. The idea is to use the pre-existent translate method. Example:

How it is:
```python
    def add_watch(self, repo, local_abspath):
        logging.debug('Adding watcher for "%s"', local_abspath)
```

How should be:
```python
    def add_watch(self, repo, local_abspath):
        logging.debug(translator['od_watcher.add_watch.add_watch.log']', local_abspath)
```

To see more examples, search for `translator` on od_pref.py

**Comment by Modelmat** [comment]
How can this be done with docstrings, and should this be done for when errors are raised?

**Comment by derrix060** [comment]
I don't think it should be done with docstrings. Docstrings are for developers only, and I think should be in English. These messages are for loggings and errors.

**Comment by Modelmat** [comment]
So for raised errors too?

**Comment by derrix060** [comment]
Yes. All messages that a normal user can view...

**Comment by derrix060** [comment]
Do you want to work on this one? I can assign it for you

**Comment by Modelmat** [comment]
Ah fudge.
Reinstalled linux and accidentally deleted all my hours of progress...
oh well.
